recipes/libtommath/recipe.yaml

```diff
@@ -0,0 +1,72 @@
+context:
+  version: "1.3.0"
+
+package:
+  name: libtommath
+  version: ${{ version }}
+
+source:
+  url: https://github.com/libtom/libtommath/archive/v${{ version }}.tar.gz
+  sha256: 6d099e93ff00fa9b18346f4bcd97dcc48c3e91286f7e16c4ac5515a7171c3149
+
+build:
+  number: 0
+  script:
+    - mkdir build
+    - cd build
+    - if: unix
+      then:
+        - cmake -G Ninja -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON ..
+    - if: win
+      then:
+        - cmake -G Ninja -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON ..
+    - ninja
+    - ninja install
+
+requirements:
+  build:
+    - ${{ compiler('c') }}
+    - ${{ stdlib('c') }}
+    - cmake
+    - ninja
+  run_exports:
+    strong:
```

Comment:
this should be `weak` because it only has to go from `host -> run` (not `build -> run`).